trace/MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 include README.rst LICENSE
-global-include *.json
+recursive-include unit_tests *
```

Comment:
For hysterical raisins. I'm fixing this in other packages now, can you update this to be `tests`?